source/core/types/ut_output_dbms_pipe.tpb

```diff
@@ -13,7 +13,11 @@ create or replace type body ut_output_dbms_pipe as
     --create an explicit private pipe. Explicitly created pipes need to be removed explicitly
     --otherwise they stay in memmory forever https://docs.oracle.com/cd/B19306_01/appdev.102/b14258/d_pipe.htm#CHDEICJI
     --to check if there are any non-purged pipes execute:  select * from v$db_pipes where pipe_size > 0 order by name desc;
+$if $$ut_pipe_enable $then
```

Comment:
Can you change the install, so that the pipe output is installed optionally for now?
Either by separate script (that still executes on Travis) or by parametrizing the install script.